documentation/book/proc-configuring-kafka-connect-replicas.adoc

```diff
@@ -5,7 +5,7 @@
 [id='proc-configuring-kafka-connect-replicas-{context}']
 = Configuring the number of broker nodes
```

Comment:
Since you are fixing this, you should probably fix the title as well ;-)

Reply:
right :-) done